doc/manifest/schema/1.1.0/installer.md

```diff
@@ -0,0 +1,479 @@
+# Windows Package Manager
+## Manifest Schema v1.1.0 Installer File
+All Windows Package Manager manifests in the Microsoft community repository are submitted using [YAML](https://yaml.org/spec/) syntax. A JSON schema is provided to aid authoring these files in editors, and in the other tooling related to the Windows Package Manager. This document provides detailed information regarding the usage of the YAML keys in the [installer](https://github.com/microsoft/winget-cli/blob/master/schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json) file for multi-file manifests. Please review the [Manifest Specification](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.1.md) if you are not familiar with this file.
+
+### [Default Locale Manifest](#tab/defaultLocale/)
+
+```YAML
+PackageIdentifier:            # The package unique identifier
+PackageVersion:               # The package version
+Channel:                      # *Not implemented* distribution channel
+Platform:                     # Optional installer supported operating system
+MimimumOSVersion:             # Optional installer minimum operating system version
+InstallerType:                # The installer type
+Scope:                        # Optional installer scope
```

Comment:
```suggestion
```
These fields are more likely to be different from installer node to installer node, so they should be included there instead of at the root level in my opinion